distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/AbstractIntegrationTest.java

```diff
@@ -735,4 +738,38 @@ public void clearCatalog() {
     public void clearCache() {
         console.runCommand(CLEAR_CACHE);
     }
+
+    public ValidatableResponse executeOpenSearch(String format, String... query) {
+        StringBuilder buffer = new StringBuilder(OPENSEARCH_PATH.getUrl()).append("?")
+                .append("format=")
+                .append(format);
+
+        for (String term : query) {
+            buffer.append("&")
+                    .append(term);
+        }
+
+        String url = buffer.toString();
+        LOGGER.info("Getting response to {}", url);
```

Comment:
Might want to update this logger to something that makes it more unique in the logs such as "Executing opensearch request to {url}". Also, does this need to be logged at an info level or a debug level?